packages/vm/lib/evm/opcodes/functions.ts

```diff
@@ -34,7 +34,7 @@ export const handlers: Map<number, OpHandler> = new Map([
   [
     0x00,
     /* eslint-disable-next-line no-unused-vars */
-    function (runState: RunState) {
+    function (/*runState: RunState*/) {
```

Comment:
Out of curiosity are both the slash-asterisk and eslint-disable comments required here?

Reply:
Hmm I didn't see it had a eslint-disable on top of it, I will re-check this file.

Reply:
Have removed these eslint-disable comments and have also removed this slash-asterisk comment stuff, d873cd9